app/renderer/js/pages/preference/base-section.js

```diff
@@ -38,6 +50,22 @@ class BaseSection extends BaseComponent {
 		}
 	}
 
+	generateDropdownOption(dropdownOption, value) {
+		if (dropdownOption === value) {
+			return `
+				<option class="dropdown-option selected" selected>
```

Comment:
You can just do, `${dropdownOption === value ? "selected" : ""}` inside the template literal here.